### PR TITLE
Add pod disruption budget

### PR DESCRIPTION
**Pull request by ncmans**

### CODE REVIEW

File: traefik/templates/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{{- if .Values.podDisruptionBudget -}}
```
Comment (steven-sheehy) [inline traefik/templates/poddisruptionbudget.yaml:1]: Would recommend an explicit `podDisruptionBudget.enabled` field so users can turn it on/off for different environments without removing default config for min/max

Comment (ncmans) [reply traefik/templates/poddisruptionbudget.yaml]: Done

**Comment by SantoDE** [comment]
Hi @ncmans,

we currently can't merge your PR, because you did not allow modifactions by the maintainers team. Could you please change that?

**Comment by ncmans** [comment]
> Hi @ncmans,
> 
> we currently can't merge your PR, because you did not allow modifactions by the maintainers team. Could you please change that?

As the fork is owned by an organization, I don't have the option to allow upstream access to PRs.

**Comment by mmatur** [comment]
@ncmans could you please bump the chart version? I will merge the PR manually.

**Pull request merged**